caa-admin/recipes/default.rb

```diff
@@ -9,6 +15,7 @@
 
 execute 'caa_admin_down' do
   command 'svc -d /etc/service/caa-admin'
```

Comment:
@mwiencek do we really need to stop the service during the update ? If the recipe fails later it will let the service stopped, which isn't great.